File: plus_ou_moins/javascript/plus_ou_moins.js
function plus_ou_moins() {
    function jeu(){
        var result = Math.floor(Math.random() * 51);
        var compteur = 1;
        while (quest != result) {
            if (quest > result) {
                quest = prompt("C'est -");
            }
            else {
                quest = prompt("C'est +");
            } 
            compteur = compteur + 1;
	       if (quest == 'quit') {
                break;
           }
        }
        if (quest == 'quit') {
            alert("Dommage, le nombre était " + result);
        }
        else {
            alert("Bravo, vous avez trouvé le nombre secret ! C'était bien " + result + " ! Tu as reussi en " + compteur + " coups !");
        }
    }
    var quest = prompt("Rentrez un nombre (entre 1 et 100) -tapez quit pour arreter");
    if (quest != 'quit'){
        jeu()
    }
}

File: plus_ou_moins/python/plus_ou_moins.py
from random import randint

while 1:
    compteur = 1
    nombre_gen = randint(0, 100)
    
    rep = int(input("Rentrez un nombre (entre 0 et 100) \n"))

    while rep != nombre_gen:
        if rep == 4242:
            break
        
        if rep < nombre_gen:
            rep = int(input("C'est + \n"))
        else:
            rep = int(input("C'est - \n"))

        compteur += 1
        
    print("Bravo, le nombre était bien " + str(nombre_gen))
    print("Tu as reussi en " + str(compteur) + " coups")
    rec = input("Recommencer ? Y/N \n")
    if rec == "Y" or rec == "y":
        continue
    else:
        break
        
